### PR TITLE
Update the table name for the 2019 data package

### PR DESCRIPTION
**Pull request by allthesignals**
This adds a migration which updates the specific sca_capacity_projects table to the latest one.

### CODE REVIEW

File: backend/db/migrate/20200507154905_update_sca_capital_projects_data_package_2019_11.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+class UpdateScaCapitalProjectsDataPackage201911 < ActiveRecord::Migration[5.2]
+  def up
+    dataPackage = DataPackage.find_by(name: 'November 2019')
+
+    schemas = dataPackage.schemas
+
+    dataPackage.schemas["sca_capacity_projects"]["table"] = "112019"
```
Comment (godfreyyeung) [inline backend/db/migrate/20200507154905_update_sca_capital_projects_data_package_2019_11.rb:7]: why alias dataPackage.schemas on line 5 but then use dataPackage.schemas again?

File: backend/db/migrate/20200507154905_update_sca_capital_projects_data_package_2019_11.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+    dataPackage.update(schemas: schemas)
+  end
+
+  def down
+    dataPackage = DataPackage.find_by(name: 'November 2019')
+
+    schemas = dataPackage.schemas
+
+    dataPackage.schemas["sca_capacity_projects"]["table"] = "022019"
```
Comment (godfreyyeung) [inline backend/db/migrate/20200507154905_update_sca_capital_projects_data_package_2019_11.rb:17]: same as above comment

**Comment by bfreeds** [comment]
@allthesignals I think when updating the data package, we need to rerun analyses that use that data package so that they incorporate the correct table in their analysis.

Believe that's what's happening here:
https://github.com/NYCPlanning/ceqr-app/blob/ab2229685a1d3e85f328b1da433385ebd05fffbd/backend/db/migrate/20191025151122_adjust_transportation_planning_factors.rb#L12

**Comment by allthesignals** [comment]
> @allthesignals I think when updating the data package, we need to rerun analyses that use that data package so that they incorporate the correct table in their analysis.
> 
> Believe that's what's happening here:
> https://github.com/NYCPlanning/ceqr-app/blob/ab2229685a1d3e85f328b1da433385ebd05fffbd/backend/db/migrate/20191025151122_adjust_transportation_planning_factors.rb#L12

Why? Wouldn't that change someone's existing analysis? That seems like a bad UX.

**Comment by allthesignals** [comment]
@bfreeds the problem with updating the seeds file is that it acts as an extension of the original database schema. Because downstream changes (migrations) depend on it being in a certain state, especially for rebuilds (which happen every time in CI), it needs to be "frozen", never touched. Ideally it's made into its own migration but for now I'm going to merge. Thanks.

**Pull request merged**